src/Kunstmaan/AdminBundle/Twig/MultiDomainAdminTwigExtension.php

```diff
@@ -24,37 +27,35 @@ public function __construct(DomainConfigurationInterface $domainConfiguration)
      */
     public function getFunctions()
     {
-        return array(
-            new \Twig_SimpleFunction('multidomain_widget', array($this, 'renderWidget'), array('needs_environment' => true, 'is_safe' => array('html'))),
-            new \Twig_SimpleFunction('is_multidomain_site', array($this, 'isMultiDomainSite')),
-            new \Twig_SimpleFunction('get_switched_host', array($this, 'getSwitchedHost')),
-            new \Twig_SimpleFunction('switched_host_is_current', array($this, 'switchedHostIsCurrent')),
-        );
+        return [
+            new Twig_SimpleFunction('multidomain_widget', [$this, 'renderWidget'], ['needs_environment' => true, 'is_safe' => ['html']]),
+            new Twig_SimpleFunction('is_multidomain_site', [$this, 'isMultiDomainSite']),
+            new Twig_SimpleFunction('get_switched_host', [$this, 'getSwitchedHost']),
+            new Twig_SimpleFunction('switched_host_is_current', [$this, 'switchedHostIsCurrent']),
+        ];
     }
 
     /**
      * Render multidomain switcher widget.
      *
      * @param \Twig_Environment $env
```

Comment:
Nitpicking here: can you remove the \

Reply:
haha, good! I hate fully qualified stuff mid code! Defining it all with use at the top, and never doing something like `new \DateTime()` is the way forward! But sure, in a docblock, you are indeed just nitpicking! :-P